examples/with-claims/api/claims/MFAClaim.ts

```diff
@@ -0,0 +1,37 @@
+import { BooleanClaim } from "supertokens-node/lib/build/recipe/session/claimBaseClasses/booleanClaim";
+
+// All of these claims/checkers are exported by us and added by endpoints of an MFA recipe
+export const SecondFactorOTPClaim = new BooleanClaim({
+    fetch: () => undefined,
```

Comment:
Why are we returning `undefined` here? Should we return `false`?

Reply:
We don't necessarily want the false info in the access token: it's not useful (we are always checking for true value) and it grows the token. Also, this was intended as a demonstration for claims only added manually in other endpoints instead of on session creation.
In an implementation where the 2fa is optional and/or the user can configure what factors they want to be enabled then we could consider adding the factor set to false as it actually contains information. (This is the same basic idea as adding a state for each factor into the token)

Reply:
Still, I don't think we'd need that info in the token.

Reply:
Alright!